src/Heartbeat/Sensor/DBUpToDate.php

```diff
@@ -2,14 +2,14 @@
 
 namespace OrcaServices\Heartbeat\Heartbeat\Sensor;
 
-use Cake\Core\Plugin;
 use Migrations\Migrations;
 use OrcaServices\Heartbeat\Heartbeat\Sensor;
 
 /**
  * DB Up to Date Sensor
  *
- * This sensor depends on the CakePHP Migrations plugin
+ * This sensor depends on the CakePHP Migrations plugin.
+ * Make sure the plugin is loaded before calling this sensor.
```

Comment:
It shouldn't be necessary with version [2.1,1](https://github.com/cakephp/migrations/releases/tag/2.1.1) of the plugin.